charts/woodpecker-agent/templates/deployment.yaml

```diff
@@ -91,3 +91,8 @@ spec:
       - name: sock-dir
         emptyDir: {}
       {{- end }}
+      {{- with .Values.dnsConfig }}
+      dnsConfig:
```

Comment:
Should we add those settings to the server chart as well?

Reply:
Since I didn't need it there I didn't add the option. Added it now though.